StarTrek13/Ships/overmap.dm

```diff
@@ -723,6 +752,35 @@ var/global/list/global_ship_list = list()
 	weapons.fire_torpedo(theturf, pilot)
 	SEND_SOUND(pilot, sound('StarTrek13/sound/borg/machines/torpedo1.ogg'))
 
+/obj/structure/overmap/ship/borg // TODO: TRACTOR BEAM, ??REGENERATIVE HULL??, ION CANNONS | A.K.A Borg balancing tool. They'll have everything needed to take a 1v1, but multiple ships will prove a difficult task to take on.
+	name = "borg cube"
+	icon_state = "borg_cube"
+	icon = 'StarTrek13/icons/trek/overmap_ships.dmi'
+	spawn_name = "BORG"
+	marker = "borg cube"
+	faction = "borg collective"
+	pixel_x = -32
+	damage = 600 // We're here to take the sheilds down, not obliterate their ship!
```

Comment:
Again, redundant, as overmap stuff autolinks. Ensure phasers are placed on the map!

Also, shields take a LOT of damage to kill.